.gitignore

```diff
@@ -7,8 +7,6 @@ lib/core/MetadataBlog.js
 
 website/translated_docs
 website/build/
-website/yarn.lock
-website/node_modules
```

Comment:
I am guessing the `node_modules` above catches this, making it redundant?

Reply:
Yes